src/bin/vip.js

```diff
@@ -27,7 +27,7 @@ const runCmd = async function () {
 	cmd
 		.command( 'logout', 'Logout from your current session' )
 		.command( 'app', 'List and modify your VIP applications' )
-		.command( 'backup', 'Generate a backup for VIP applications' )
+		.command( 'backup', 'Generate a backup of an environment.' )
```

Comment:
This change matches what I said above in `src/bin/vip-backup.ts` so we should probably make it match. Also, should they all say "fresh" or "new" or neither?

Reply:
"New" is preferred to "fresh". Thank you for picking up on these details!